src/utils/ast-helpers.ts

```diff
@@ -142,15 +146,15 @@ export function findClassPropertiesConstructorParameterByType(node: ClassDeclara
 }
 
 export function findClassPropertiesDeclarationByType(node: ClassDeclaration, type: string): string[] {
-	const query = `PropertyDeclaration:has(TypeReference > Identifier[name="${type}"]) > Identifier`;
-	const result = tsquery<Identifier>(node, query);
-	return result.map((n) => n.text);
+	const query = `PropertyDeclaration:has(TypeReference > Identifier[name="${type}"])`;
```

Comment:
To make it capture both `Identifier` and `PrivateIdentifier` I reduced the specificity of the query and then extract the identifier from the AST instead of getting it directly from the query.